lib/tests/formulae_detect.rb

```diff
@@ -42,8 +43,8 @@ def detect_formulae!(args:)
           @testing_formulae = []
           # Use GitHub Actions variables for pull request jobs.
           if github_ref.present? && github_repository.present? &&
-             %r{refs/pull/(?<pr>\d+)/merge} =~ github_ref
-            url = "https://github.com/#{github_repository}/pull/#{pr}/checks"
+             %r{refs/pull/(?:\d+)/merge} =~ github_ref
```

Comment:
```suggestion
             %r{refs/pull/(\d+)/merge} =~ github_ref
```
or else `Regexp.last_match(1)` will always be `nil`.